api/src/main/resources/workbench.yaml

```diff
@@ -727,6 +727,62 @@ paths:
           schema:
             $ref: '#/definitions/EmptyResponse'
 
+  /v1/workspaces/{workspaceNamespace}/{workspaceId}/share:
```

Comment:
These were grouped with the Cron endpoints for some reason

Reply:
I believe @als364 has a tool to rearrange the swagger file. Alphabetizing the paths and definitions would be awesome.

Reply:
I don't have a tool per se I just want to alphabetize it. Maybe next Friday in between all the meetings.